Source/WebKit/Shared/RemoteLayerTree/CGDisplayListImageBufferBackend.cpp

```diff
@@ -108,7 +109,9 @@ ImageBufferBackendHandle CGDisplayListImageBufferBackend::createBackendHandle(Sh
     RELEASE_LOG(RemoteLayerTree, "CGDisplayListImageBufferBackend of size %dx%d encoded display list of %ld bytes", size.width(), size.height(), CFDataGetLength(data.get()));
 #endif
 
-    return ImageBufferBackendHandle { IPC::SharedBufferReference { WebCore::SharedBuffer::create(data.get()) } };
+    CGDisplayList displayList;
+    displayList.m_displayList = WebCore::SharedBuffer::create(data.get());
```

Comment:
Why set a private data member like this instead of passing the display list to the constructor?

Reply:
Fixed! This was a silly thing I inherited from ShareableBitmap::Handle.